velox/exec/tests/CprHttpClientTest.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/tests/GTestUtils.h"
+
+#include <cpr/cpr.h>
+#include <map>
+
+class CprHttpClientTest : public testing::Test {};
+
+TEST_F(CprHttpClientTest, basic) {
+  auto response = cpr::Get(
+      cpr::Url{"https://facebookincubator.github.io/velox/"},
```

Comment:
Internally when we run these tests I am not sure that they will be able to access the general internet (This might be false so I will double check).